openpype/hosts/resolve/api/plugin.py

```diff
@@ -408,7 +408,11 @@ def load(self, files):
         _clip_property = media_pool_item.GetClipProperty
         source_in = int(_clip_property("Start"))
         source_out = int(_clip_property("End"))
-        source_duration = int(_clip_property("Frames"))
+
+        # Trim clip start if slate is present
+        if "slate" in self.data["versionData"]["families"]:
+            source_in += 1
+        source_duration = source_out - source_in + 1
```

Comment:
Why would this be more reliable then the `Frames` property? Couldn't we do `source_duration -= 1` for the `slate` case? :)